src/helpers/invert.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import parseColorString from '../internalHelpers/_parseColorString'
+import rgb from './rgb'
+import rgba from './rgba'
+
+/**
+ * Returns the inverted color meaning the red, green and blue values are invert.
```

Comment:
That's a pretty redundant description 😉 What about `"Inverts the red, green and blue values of a color"` or something like that?